contracts/land/LANDRegistry.sol

```diff
@@ -50,6 +51,10 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     _;
   }
 
+  //
+  // LAND Create and destroy
```

Comment:
I think this comment does not belong here. How can we destroy LANDs?

Reply:
Should be removed it's an old comment